package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "type": "module",
+  "name": "call_of_duty",
+  "version": "1.0.0",
+  "description": "Call Of Duty",
+  "main": "server.js",
+  "scripts": {
+    "test": "logLevel=silent vitest",
+    "start": "node --experimental-specifier-resolution=node server.js | pino-pretty",
```

Comment:
experimental is not good if you don't have good reason...
Add files extension to imports as [needed](https://nodejs.org/api/esm.html#mandatory-file-extensions)

Reply:
done